src/components/dropdowns/primary-action-dropdown/primary-action-dropdown.spec.js

```diff
@@ -142,8 +143,10 @@ describe('when dropdown is open and body is clicked', () => {
     expect(getByLabelText('Secondary Action')).toBeInTheDocument();
 
     // click on doucment body
-    document.body.click();
 
+    act(() => {
```

Comment:
needed this or else got warnings in the test

Reply:
Cool that it worked.

Reply:
Since RTL already includes this, can you maybe try to click on the rendered container?

```js
fireEvent.click(rendered.container)
```

There should be no need for using `act` directly

Reply:
Fixed here: https://github.com/commercetools/ui-kit/pull/965/commits/b83d74f503cd5d44322219ca1b0b6b2a7c2f624f